x/cash/staticfee.go

```diff
@@ -1,3 +1,18 @@
+/*
+
```

Comment:
Package-level godoc is nice.
Maybe we can move some text for the Handler there as well?

Reply:
What do you mean? I cannot find any other big chunk of comment in the `x/cash` package.